io500.sh

```diff
@@ -137,31 +131,13 @@ function run_benchmarks {
   source ./utilities/io500_fixed.sh 2>&1 | tee $io500_result_dir/io-500-summary.$timestamp.txt
 }
 
-# Add key/value pairs defining your system
-# Feel free to add extra ones if you'd like
+# Information fields; these provide information about your system hardware
+# Use https://vi4io.org/io500-info-creator/ to generate information about your hardware
```

Comment:
We said we wouldn't make this change until the info creator had pull-downs.

Reply:
I am working on this, I have to fix some bugs.